simplejson/_speedups.c

```diff
@@ -76,6 +76,9 @@ static PyObject *JSON_Infinity = NULL;
 static PyObject *JSON_NegInfinity = NULL;
 static PyObject *JSON_NaN = NULL;
 static PyObject *JSON_EmptyUnicode = NULL;
+#if PY_MAJOR_VERSION < 3
+static PyObject *JSON_EmptyStr = NULL;
```

Comment:
Is `JSON_EmptyUnicode` still used in Python 2? If no, then two variables can be merged.

Reply:
Yes, it is used on all Python versions.